bin/export-db-to-sqlite.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -xe
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Exports the postgres database to a sqlite file, which can be copied to the
+# cloud bucket for consumption by web pods.
+
+# Usage:
+# ./bin/export-db-to-sqlite.sh /path/to/output.db
+
+# CRITICAL: requires DATABASE_URL in the environment (or temporarily set at the
+# time of running), pointing to the source *Postgres* DB
+
+output_db=$1
+
+all_well=true
+
+check_status() {
+    if [[ $all_well == false ]]; then
+        echo "ERROR: there was a problem with the export process. Error code: $1"
+        exit $1
+    fi
+}
+
+# 1. Dump out to json from the default, source DB
+
+python manage.py dumpdata \
+    contenttypes \
+    --indent 2 \
+    --natural-primary \
+    --natural-foreign \
+    --output /tmp/export_contenttypes.json || all_well=false
+
+python manage.py dumpdata \
+    wagtailcore.Locale \
+    --indent 2 \
+    --natural-foreign \
+    --output /tmp/export_wagtail_locales.json || all_well=false
+
+
+# Noted exclusions:
+# * Sessions are transient and not needed
+# * ContentTypes are dumped separately
+# * Wagtail Locales are dumped separately
+# * PageSubscription is based on User data, which is not ported over
+# * Renditions are not needed as they can be regenerated
+# * ReferenceIndex is not needed as it can be regenerated
+# * WagtailSearch indices need rebuilding and search history is not important
+
+
+python manage.py dumpdata \
+    --natural-foreign \
+    --exclude=contenttypes \
+    --exclude=sessions.Session \
+    --exclude=wagtailcore.Locale \
+    --exclude=wagtailcore.PageSubscription \
+    --exclude=wagtailimages.Rendition \
+    --exclude=wagtailcore.ReferenceIndex \
+    --exclude=wagtailsearch \
+    --indent 2 \
+    --output /tmp/export_remainder.json || all_well=false
+
+check_status 99
+
+# 2. Prep a fresh sqlite DB with schema, deleting the original
+rm $output_db
```

Comment:
I think this will blow up if the `$output_db` doesn't already exist, right? probably best to `rm -f` here?

Reply:
Yeah, good call.